src/ucp/core/ucp_request.inl

```diff
@@ -670,4 +671,55 @@ ucp_request_param_mem_type(const ucp_request_param_t *param)
            param->memory_type : UCS_MEMORY_TYPE_UNKNOWN;
 }
 
+static UCS_F_ALWAYS_INLINE ucs_ptr_map_key_t
+ucp_request_get_key(ucp_worker_h worker, ucp_request_t *req, int indirect)
+{
+    ucs_ptr_map_t *ptr_map = &worker->ptr_map;
+    ucs_ptr_map_key_t key;
+    ucs_status_t status;
+
+    status = ucs_ptr_map_put(ptr_map, req, indirect, &key);
+    if (ucs_unlikely(indirect)) {
```

Comment:
minor: can do shorter, like
```
if (ucs_unlikely(status != UCS_OK)) {
    ucs_assert(indirect);
    key = UCP_WORKER_PTR_MAP_KEY_INVALID;
}
```

Reply:
I agree it's shorter but intention is do not check return value for "direct" case

Reply:
ok